integration/client.go

```diff
@@ -41,6 +41,20 @@ func (c *HyperClient) GetPodInfo(podID string) (*types.PodInfo, error) {
 	return pod.PodInfo, nil
 }
 
+// ExitCode gets exitcode by container and tag
+func (c *HyperClient) ExitCode(container, tag string) (int32, error) {
+	request := types.ExitCodeRequest{
+		Container: container,
+		Tag:       tag,
```

Comment:
ExitCode is for process(container's init process or exec process). the tag will be replaced by process Id.